wire-ios/Tests/TestPlans/SecurityTests.xctestplan

```diff
@@ -137,6 +137,32 @@
         "identifier" : "BACB88501AF7C48900DDCDB0",
         "name" : "Wire-iOS-Tests"
       }
+    },
+    {
+      "skippedTests" : [
+        "AnimatedListMenuViewTests",
+        "AuthenticationEmailVerificationRequiredErrorHandlerTests",
+        "AvailabilityStringBuilderTests",
+        "CountryTests",
+        "HandleChangeStateTests",
+        "NetworkConditionHelperTests",
+        "OrientationDeltaTests",
+        "PhoneNumberTests",
+        "StringCapitalizationTests",
+        "StringWholeRangeTests",
+        "String_PhoneNumberTests",
+        "TemporaryFileServiceTests",
+        "TransformTextViewTests",
+        "UITextView_ReplaceTests",
+        "UITraitEnvironmentTests",
+        "UserRightTests",
+        "ZMUserMentionsTest"
```

Comment:
why are these skipped? the SecurityTests plan should not have the option to automatically add new tests. So we explicitly add tests